app/src/main/java/com/android/brewr/utils/FetchNearbyCoffeeShops.kt

```diff
@@ -9,7 +9,7 @@ import com.android.brewr.BuildConfig
 import com.android.brewr.model.coffee.Coffee
 import com.android.brewr.model.coffee.Hours
 import com.android.brewr.model.coffee.Review
-import com.android.brewr.model.location.Location
+import com.android.brewr.model.map.Location
```

Comment:
This fix was very sharp. We almost have confused that we have two "location" files. Thanks to you we will use the correct import !

Reply:
Thanks!!